### PR TITLE
[Bugfix:TAgrading] Peer grading team interface fixes

### PR DESCRIPTION
**Pull request by jchulton**
### Please check if the PR fulfills these requirements:

* [ ] The PR title and message follows our [guidelines](https://submitty.org/developer/how_to_contribute)
* [ ] Tests for the changes have been added/updated (if possible)
* [ ] Documentation has been updated/added if relevant

### What is the current behavior?
Peer team rotating section assignments will produce a robot when a student tries to grade them.

Students can see all peer assignments in the navigation menu if they are assigned to grade one peer assignment

### What is the new behavior?
Fixes these errors: peer team rotatin section assignments can be graded properly, students can only see gradeables they are assigned to grade in the navigation menu.

### CODE REVIEW

File: site/app/libraries/database/DatabaseQueries.php [hunk 1]
```diff
@@ -1744,6 +1744,11 @@ public function getRotatingSectionsForGradeableAndUser($g_id, $user_id = null) {
     public function getUsersByRotatingSections($sections, $orderBy = "rotating_section") {
         $return = array();
         if (count($sections) > 0) {
+            $orderBy = str_replace(
+                "Peer",
+                1,
+                $orderBy
+            );
```
Comment (MasterOdin) [inline site/app/libraries/database/DatabaseQueries.php:1751]: Why was this necessary? Where this function being called that the `$orderBy` parameter has the word "Peer" in it?

Comment (jchulton) [reply site/app/libraries/database/DatabaseQueries.php:1751]: On line 526 of ElectronicGraderController.php. For team peer graders, we named the rotating section Peer so that it could be differentiated from non-peer rotating sections. There are other ways we could differentiate that I am working on

Comment (MasterOdin) [reply site/app/libraries/database/DatabaseQueries.php:1751]: Is that change in some future PR that you and @emaicus are working on as line 526 of ElectronicGraderController.php is just this:
https://github.com/Submitty/Submitty/blob/c9873ecbe67a9129f62450ed134034e60dd72902/site/app/controllers/grading/ElectronicGraderController.php#L526

where there is no second parameter to the function call...
